### DISCUSSION TITLE
Revert to "For you" and "Following" feeds

### DISCUSSION THREAD

**Discussion by icebarf** [community/community#66354] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

As the title specifies, separation of the feeds is something I am up for. This allows me to keep tabs on repositories, and people I follow without getting in my way. The "oldest" feed was only for the people a user followed, paired with options like "watching" a repository. I personally prefer the separated feeds.

**Comment by seefun** [comment]
+1
